app/views/deploys/active.erb

```diff
@@ -6,11 +6,16 @@
   </section>
   <%= link_to 'job execution debug info', '?debug=1' %>
 <% else %>
-  <% active, queued = JobQueue.debug %>
+  <% active, queued, staggered = JobQueue.debug %>
   <section class="tabs">
     <h2>Active</h2>
     <%= render 'job_executions', job_executions: active %>
 
+    <% if staggered %>
+      <h2>Waiting to Start</h2>
```

Comment:
makes contribution easier if UI and code are in sync -> `Staggered (waiting to start)` or `Waiting to start (Staggered)`